CMakeLists.txt

```diff
@@ -57,4 +60,8 @@ add_executable(${CMAKE_PROJECT_NAME} ${PROJECT_SOURCES})
 target_link_libraries(${CMAKE_PROJECT_NAME} PUBLIC ${LINK_LIBS})
 
 list(APPEND CMAKE_MODULE_PATH ${PINCHOT_API_ROOT_DIR})
-include(PinchotBuildApplication)
+include(PinchotBuildApplication RESULT_VARIABLE HAVE_PINCHOT_BUILD_APP)
+#Could get rid of this since include already throws an error if it cant find the cmake file but adds some clarity
+if(NOT HAVE_PINCHOT_BUILD_APP)
+  message("PinchotBuildApplication was not found meaning PINCHOT_API_ROOT_DIR was likely not set propperly")
```

Comment:
There's a typo here "propperly" should be "properly". Also, when writing user facing messages, there's a certain "style" that I think is good to maintain; this is something that one only really picks up through experience and is a learning process (I'm still learning myself). I would recommend something like:

>PinchotBuildApplication was not found, check PINCHOT_API_ROOT_DIR define